SampleCode/Python/sample_identify_cross_page_tables.ipynb

```diff
@@ -230,7 +491,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "identify_cross_page_tables()      \n"
+    "if __name__ == \"__main__\":\n",
+    "    # Check if the input string is provided as a command-line argument\n",
+    "    if len(sys.argv) > 1:\n",
+    "        input_file_path = sys.argv[1]\n",
+    "    else:\n",
+    "        print(\"Usage: python sample_identify_cross_page_tables.py [input_file_path]\")\n",
+    "        sys.exit(1)\n",
+    "\n",
+    "    identify_and_merge_cross_page_tables(input_file_path)"
    ]
```

Comment:
how do you run this in the ipynb? My approach is just call the function directly "identify_and_merge_cross_page_tables(input_file_path)". maybe I'm not aware of this approach in jupyter notebook.

Reply:
Fixed. Thx~
![JupyterLabScreen](https://github.com/microsoft/Form-Recognizer-Toolkit/assets/26052497/cd8a7106-0e33-43b5-8800-e12abd435359)